.env

```diff
@@ -1,3 +1,11 @@
 PORT=65534
 NODE_ENV=development
-MONGO_CONNECTION=mongodb+srv://qwerty:1234qwer@cluster0.5qjjt.mongodb.net/netflix
+MONGO_CONNECTION=mongodb+srv://qwerty:1234qwer@cluster0.njhap.mongodb.net/todolist-be
+JWT_REFRESH_SECRET=jwt-refresh-secret-key
+JWT_ACCESS_SECRET=jwt-secret-key
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=465
+SMTP_USER=dr.savaug.testing@gmail.com
+SMTP_PASSWORD=pufho9-jathyd-wEwhuj
```

Comment:
nobody commits .env file to repo, especially with secrets

Reply:
ok.